readme.rst

```diff
@@ -3,63 +3,40 @@ Documentation starter pack
 
 See the `Sphinx and Read the Docs <https://canonical-documentation-with-sphinx-and-readthedocscom.readthedocs-hosted.com/>`_ guide for instructions on how to get started with Sphinx documentation.
 
-Then go through the following sections to use this starter pack to set up your docs repository.
+Then go through the following sections to use this starter pack to set up your documentation repository.
 
-Set up your documentation repository
-------------------------------------
+Prerequisites
+-------------
 
-You can either create a standalone documentation project based on this repository or include the files from this repository in a dedicated documentation folder in an existing code repository.
+Before you begin, ensure you have the following:
 
-**Note:** We're planning to provide the contents of this repository as an installable package in the future, but currently, you need to copy and update the required files manually.
+ * A GitHub repository where you want to host your documentation. The recommended approach is to host the documentation alongside your code in a `docs` folder. But a standalone documentation repository is also an option.
+ * Git and Bash installed on your system.
 
-Standalone documentation repository
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Bootstrap your documentation repository
+---------------------------------------
 
-To create a standalone documentation repository, clone this starter pack
-repository, `update the configuration <#configure-the-documentation>`_, and
-then commit all files to the documentation repository.
+Bootstrap your repository by following these steps:
 
-You don't need to move any files, and you don't need to do any special
-configuration on Read the Docs.
+ * Copy [bootstrap.sh](bootstrap.sh) to your repository's root directory.
```

Comment:
The Markdown link doesn't work - convert to RST. 
Not entirely sure about the syntax; my guess would be either `` `bootstrap.sh`_ `` or `` :download:`bootstrap.sh` ``.

Reply:
I just removed the link. The readme doc exists in two places - on GitHub and in the built starter pack. On GitHub, a relative link works fine. In the built doc, it doesn't. I could try add it to the doc, and add to `index.rst`, but then, what is a `.sh` file going to look like in the built doc?